plugin/src/main/kotlin/tanvd/grazi/grammar/GrammarEngine.kt

```diff
@@ -50,9 +50,9 @@ object GrammarEngine {
     private fun getTyposSmall(str: String): LinkedSet<Typo> {
         if (isSmall(str)) return LinkedSet()
 
-        val lang = LangDetector.getLang(str, GraziConfig.get().enabledLanguages.toList()) ?: return LinkedSet()
+        val lang = LangDetector.getLang(str, GraziConfig.get().availableAndEnabledLanguages.toList()) ?: return LinkedSet()
 
-        return tryRun { LangTool[lang].check(str) }
+        return tryRun { LangTool[lang]!!.check(str) }
```

Comment:
Let's think about logging here ...

Reply:
Well, can you add an exception log in this function in your lib?

Reply:
Or add an additional argument - errorHandler with default null

Reply:
Add try catch